zgw/catalogi-api/gradle/publishing.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pluginManager.withPlugin('maven-publish') {
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                pom {
+                    name = 'Catalogi API module'
+                    description = 'This module facilitates working with a Catalogi API compliant microservice'
+                    developers {
+                        developer {
+                            id = "team-valtimo"
```

Comment:
@ThomasMinkeRitense Do we want to continue using 'valtimo' in the portal?

Reply:
We don't have something better yet, so leave it in for now.